requirements/dev.txt

```diff
@@ -89,8 +89,8 @@ snowballstemmer==2.0.0    # via pydocstyle
 sqlparse==0.3.1           # via django
 stevedore==1.32.0         # via edx-opaque-keys
 text-unidecode==1.3       # via faker
-toml==0.10.1              # via tox
```

Comment:
these .txt files are automatically generated based on what is in the corresponding .in files.  If you want to pin a version you would need to do it there.  You can find tox in the travis.in file.  However, we should only be preventing updates if there's a good reason our code cannot be compatible with future versions of the library or, we have a task to solve it at some point in the future.

Do we know why tox can't be bumped to 0.10.1?